### ISSUE TITLE
Not reading Code 128B format codes

### ISSUE THREAD

**Issue by mbugbee** [Redth/ZXing.Net.Mobile#973]
Does this library support the different subtypes of Code 128?

**Comment by thhowl** [comment]
Yes, library version 2 and 3 support Code 128A, B, and C, when you specify Code 128:

`ScannerView.Options.PossibleFormats = new []{ZXing.BarcodeFormat.CODE_128};`

I have confirmed this using barcodes at this site: https://www.softmatic.com/barcode-code-128.html
